src/ga/ga_population.rs

```diff
@@ -90,18 +99,31 @@ impl<T: GAIndividual> GAPopulation<T>
         self.sort_order
     }
 
+    pub fn set_order_and_sort(&mut self, order: GAPopulationSortOrder)
+    {
+        // TODO: Test that changing order after the population has been used doesn't
+        // cause problems.
```

Comment:
I mean, you could make the point of being unsafe as you can kinda screw the whole point of the algorithm (if you just flip flop the orders, you get worse than random search).
But... it's pretty explicit, so if you call it is your fault. I guess the question would be how necessary is for current use cases?